### DISCUSSION TITLE
SCIM provisioning support added

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#4598] [Ideas]
[A customer](https://app.hubspot.com/contacts/3888472/contact/19299101) 🔒  asks:

> Can you confirm if Parabol supports SCIM or a provisioning API?

Opening this as a discussion until we have AC nailed down.

See: http://www.simplecloud.info/

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/28083851) 🔒 says:

> Do you have a provisioning API?

**Comment by drewhousman1010** [comment]
A[ customer](https://app.hubspot.com/contacts/3888472/contact/67299501) 🔒 asks:

> do you support JIT Provisioning & Deprovisioning with your SAML integration on the enterprise plan?

**Comment by drewhousman1010** [comment]
A [longtime enterprise customer](https://app.hubspot.com/contacts/3888472/record/0-1/67283401) 🔒  says:

> Any update on APIs for user management? i.e. to deactivate?
> 
> We use SAML, but would love to disable on deactivation right away as well on your end.

**Comment by drewhousman1010** [comment]
A [new Enterprise customer](https://app.hubspot.com/contacts/3888472/record/0-1/138806759081) 🔒 asks: 

> Is SCIM provisioning enabled in the platform? And if so, could they provide the documentation to set this up?
